src/main.rs

```diff
@@ -114,6 +114,18 @@ fn create_app<'a, 'b>(
                 .default_value(&default_memory)
                 .group("vm-config"),
         )
+        .arg(
+            Arg::with_name("memory-region")
+                .long("memory-region")
+                .help(
+                    "Custom memory region parameters \
+                     \"size=<guest_memory_region_size>,mergeable=on|off,\
+                     shared=on|off,hugepages=on|off\"",
```

Comment:
I am assuming the use of `--memory-region` option will be mostly helpful when users specify explicit backing files. If that's true, do we want to expose the `file` parameter explicitly here?

Reply:
Oh yes I missed that. I'll fix it.